### ISSUE TITLE
[OH3] Config appears to be double escaped when set from UI

### ISSUE THREAD

**Issue by MikeJMajor** [openhab/openhab-core#1779]
Not sure if this is core, UI or user error issue.

While developing a serial binding I spotted that I wasn't sending new line characters over the serial interface.

With the following thing defined in configuration it works as I expect, if I send an UP command to the 'serialRollo' channel it sends 'Rollo_UP' followed by a LF to the serial port

```
Thing serialDevice rollershutter [patternMatch=".*"] {
        Channels:
            Type rollershutter : serialRollo [transform="REGEX(Position:([0-9.]*))", up="Rollo_UP\n", down="Rollo_DOWN\n", stop="Rollo_STOP\n"]
            Type switch : roloAt100 [transform="REGEX(s/Position:100/ON/)"]
    }
```
If I configure the thing in the UI and set the channel config parameter 'up' to 'Rollo_UP\n' it sends 'Rollo_UP\n' with no LF.

I'm guessing that its being JSON escaped when sent from the UI and again JSON escaped when stored in the JSON db.

**Comment by adamgoldstein** [comment]
I suspect I'm having the same issue. Mike, were you able to find a workaround for this issue or do I need to move my Things from PaperUI to files?

**Comment by MikeJMajor** [comment]
@ghys do you have any thoughts on this one?

**Comment by ghys** [comment]
Yes a regular text box doesn't support escape characters, if you type some they're treated as literals and therefore automatically escaped.

![image](https://user-images.githubusercontent.com/2004147/120644120-443e0680-c477-11eb-8231-2dfd4648a471.png)

Only textareas can display multiline strings:

![image](https://user-images.githubusercontent.com/2004147/120645057-4eacd000-c478-11eb-85a8-fa1883edc0b8.png)
